### PR TITLE
Christian Anselm

### PR DESCRIPTION
**Pull request by ChristianAnselm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,109 @@
 # Loop Exercises
 
 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
-2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num". 
+
+let num = 11;
+while (num > 1) {
```
Comment (crymall) [inline README.md]: This wouldn't log 1.

File: README.md [hunk 2]
```diff
@@ -1,26 +1,109 @@
...
+
+
+
+3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num".
 For each iteration of the loop, it will multiply the number by 9 and log the result (e.g. "2 * 9 = 18").
 
-_Bonus_ think of another way to solve it. 
+let num = 0;
+while(num <= 10) {
+  console.log( num * 9)
```
Comment (crymall) [inline README.md]: The only thing here is that you're supposed to log the entire math equation. What you're doing mathematically and loop-wise, however, is great :)

**Comment by crymall** [comment]
This is great - well-formatted code, and it's clear your understanding of loops is strong. Great work!

File: README.md (hunk 1 above)
Comment (ChristianAnselm) [reply README.md]: while (num <= 1)

**Pull request closed without merging**